CHANGES.md

```diff
@@ -1,3 +1,7 @@
+# 2.9.4
+- [NOTE] Add `axios` to peerDependencies.
+- [UPGRADED] `@ibm-cloud/cloudant` dependency to version `0.2.1`.
```

Comment:
nit, we normally list `upgraded` items before `note` items.

Reply:
thanks, fixed